README.md

```diff
@@ -31,12 +57,221 @@ const structure = [
 />;
 ```
 
-## Props
+ ## Material tree
+ 
+  <p align="center">
+      <img src="react-tree-demo.gif?raw=true" width="400" />
+  </p>
+  
+  ### Installation
+  
+   **Installation using npm:**
+  ```
+   npm install @kenshooui/material-tree --save
+  ```
+ **Installation using Yarn:**
+
+```
+ yarn add @kenshooui/material-tree
+```
+
+ ### How to use
+ <!-- example -->
+
+```jsx
+import MaterialTree from "@kenshooui/material-tree";
```

Comment:
This is the root readme, it should not contain the readme for the material design.
You have the root readme that should discuss the core component and link to a readme for the different themes.
In those specific theme readme you can show information on how to use the theme.

Reply:
@liorheber What about the core readme?
Should I describe all the information related to it in the root readme of the project? or in the inner readme of the core package?